tests/accuracy_validation/test_accuracy_validation.py

```diff
@@ -59,11 +59,17 @@
 
 @pytest.fixture(autouse=True, scope="module")
 def setup_ac():
-    subprocess.run('virtualenv -ppython3.8 acc_check', cwd=PROJECT_ROOT, check=True, shell=True)
-    subprocess.run(f'{acc_check_activate_string} && {ACC_CHECK_VENV_DIR}/bin/python setup.py install',
+    subprocess.run('virtualenv -ppython3.7 acc_check', cwd=PROJECT_ROOT, check=True, shell=True)
```

Comment:
Pavel, probably it would be better:
```suggestion
    subprocess.run(f'virtualenv -ppython3.7 {ACC_CHECK_VENV_DIR}', cwd=PROJECT_ROOT, check=True, shell=True)
```
-- it should work, shouldn't it?

Reply:
Probably it would be better to use
```diff
- TEST_ROOT = Path(__file__).parent
+ TEST_ROOT = Path(__file__).parent.resolve()
```
-- I like using absolute paths everywhere, even if the`__file__` is a relative path

Reply:
> Pavel, probably it would be better:
> 
> -- it should work, shouldn't it?

Done - see Build 295